pages/settings/wallets/[wallet].js

```diff
@@ -67,10 +67,16 @@ export default function WalletSettings () {
     }
   }, [wallet.def])
 
+  const { card: { image, title, subtitle } } = wallet?.def || { card: {} }
+
   return (
     <CenterLayout>
-      <h2 className='pb-2'>{wallet?.def.card.title}</h2>
-      <h6 className='text-muted text-center pb-3'><Text>{wallet?.def.card.subtitle}</Text></h6>
+      {image
+        ? typeof image === 'object'
+          ? <img {...image} alt={title} className='pb-2' />
+          : <img src={image} width='33%' alt={title} className='pb-2' />
```

Comment:
I'm not sure this img path thing will work in prod ... they might need to go in `./public`.